Maven plugin add support for jar projects in `:effective-web-xml`

I'm building an app as a simple jar project, with embedded Jetty included as a dependency. I've found out that Jetty supports the quickstart mode, thus vastly speeding up boot time (and possibly even enabling native mode with GraalVM).

Unfortunately, `mvn jetty:effective-web-xml` fails with `Packaging type [jar] is unsupported` which leads me to believe that only the WAR-type projects are supported at the moment.

However, the jar-type project has a classpath very similar to the war-type project, and so this ticket could perhaps be fixed simply by removing the limitation in the Jetty Maven Plugin.

Thanks again for your great work :)

I'm using Jetty 11.0.14.

Unfortunately, the plugin fails with:
```
Caused by: java.lang.NullPointerException: Cannot invoke "java.nio.file.Path.getName(int)" because "path" is null
    at org.eclipse.jetty.maven.plugin.JettyEffectiveWebXml.configureWebApp (JettyEffectiveWebXml.java:66)
    at org.eclipse.jetty.maven.plugin.AbstractWebAppMojo.startJetty (AbstractWebAppMojo.java:430)
    at org.eclipse.jetty.maven.plugin.AbstractWebAppMojo.execute (AbstractWebAppMojo.java:419)
```

Indeed, `path` is null since `war` is most probably null since we're not having a war app: https://github.com/eclipse/jetty.project/blob/jetty-11.0.x/jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyEffectiveWebXml.java#L66

--- a/jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyEffectiveWebXml.java
+++ b/jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyEffectiveWebXml.java
@@ -63,12 +63,11 @@ public class JettyEffectiveWebXml extends AbstractUnassembledWebAppMojo
             }
         }
 
-        Path start = path.getName(0);
-        int count = path.getNameCount();
-        Path end = path.getName(count > 0 ? count - 1 : count);
-        //if the war is not assembled, we must configure it
-        if (start.startsWith("src") || !end.toString().endsWith(".war"))
+        
+        if ((path == null) || (path.startsWith("src") || !path.endsWith(".war")))
+        {
             super.configureUnassembledWebApp();
+        }
     }
     
     /**

